.eslintignore

```diff
@@ -3,5 +3,9 @@ typechain-types
 artifacts
 cache
 coverage
+hardhat.config.d.ts
+hardhat.config.js
 network.d.ts
 network.js
+scripts/*.js
+scripts/*.d.ts
```

Comment:
Not sure why these files didn't need to be ignored by the linter initially, but adding them now.